torch/_inductor/scheduler.py

```diff
@@ -88,7 +88,9 @@ def __init__(self, scheduler: "Scheduler", node: ir.Buffer):
         self.ancestors: Optional[Set[str]] = None
         self.min_order: Optional[int] = None
         self.max_order: Optional[int] = None
```

Comment:
I guess we are declaring `min_order` and `max_order` here (even though the BaseSchedulerNode methods don't use them) because we want to be able to access them on instances of BaseSchedulerNode. But it doesn't seem like we ever actually expect them to have a `None` value. I *think* we could hack around this by declaring uninitialized fields, similar to what I did here: https://github.com/pytorch/pytorch/blob/fa7e5170f3c0b37552921fac6516aa513db3733d/torch/_inductor/ir.py#L309-L322

Reply:
yeah makes sense.